Source/WebCore/platform/graphics/displaylists/DisplayListRecorder.cpp

```diff
@@ -558,6 +558,7 @@ void Recorder::resetClip()
     currentState().clipBounds = m_initialClip;
 
     recordResetClip();
+    clip(m_initialClip);
```

Comment:
The currentState's clip bounds was already set to `m_initialClip` on line 558 so this seems like an appropriate change.